### PR TITLE
GNN shooting at test time

### PR DESCRIPTION
**Pull request by ronuchit**

### CODE REVIEW

File: tests/envs/test_pybullet_utils.py [hunk 1]
```diff
@@ -18,7 +18,7 @@ def _setup_pybullet_test_scene():
 
     Initialized only once and cached globally for efficiency.
     """
-    global _PYBULLET_SCENE_IS_SETUP  # pylint:disable=global-statement
+    global _PYBULLET_SCENE_IS_SETUP  # pylint: disable=global-statement
```
Comment (ronuchit) [inline tests/envs/test_pybullet_utils.py:21]: @tomsilver can we use the fixture here too :)

Comment (tomsilver) [reply tests/envs/test_pybullet_utils.py:21]: Yup will do tomorrow

Comment (ronuchit) [reply tests/envs/test_pybullet_utils.py:21]: #554

**Comment by ronuchit** [comment]
Results, reusing the trained models from before:
```
Git commit hashes seen in results/:
04bb6d8e8612ebdf202465a3673f4fa050feb1b6


AGGREGATED DATA OVER SEEDS:
                         NUM_TRAIN_TASKS     NUM_SOLVED AVG_NUM_PREDS AVG_TEST_TIME AVG_NODES_CREATED    LEARNING_TIME  NUM_SEEDS
EXPERIMENT_ID
blocks_gnn_exclude_100     100.00 (0.00)   21.00 (7.20)   0.00 (0.00)   0.32 (0.27)       0.00 (0.00)   305.12 (73.49)         10
blocks_gnn_exclude_1000   1000.00 (0.00)   32.40 (8.48)   0.00 (0.00)   0.44 (0.31)       0.00 (0.00)   195.27 (87.88)         10
blocks_gnn_exclude_150     150.00 (0.00)   26.70 (8.87)   0.00 (0.00)   0.36 (0.24)       0.00 (0.00)   250.22 (85.06)         10
blocks_gnn_exclude_200     200.00 (0.00)  27.00 (10.50)   0.00 (0.00)   0.36 (0.20)       0.00 (0.00)  245.61 (106.32)         10
blocks_gnn_exclude_50       50.00 (0.00)   20.60 (8.98)   0.00 (0.00)   0.43 (0.26)       0.00 (0.00)   312.73 (85.22)         10
blocks_gnn_exclude_500     500.00 (0.00)   31.20 (9.45)   0.00 (0.00)   0.35 (0.25)       0.00 (0.00)   203.57 (95.57)         10
cover_gnn_exclude_100      100.00 (0.00)   49.50 (1.50)   0.00 (0.00)   0.54 (0.17)       0.00 (0.00)    31.92 (20.87)         10
cover_gnn_exclude_1000    1000.00 (0.00)   50.00 (0.00)   0.00 (0.00)   0.37 (0.08)       0.00 (0.00)     18.69 (3.76)         10
cover_gnn_exclude_150      150.00 (0.00)   50.00 (0.00)   0.00 (0.00)   0.39 (0.07)       0.00 (0.00)     19.79 (3.67)         10
cover_gnn_exclude_200      200.00 (0.00)   50.00 (0.00)   0.00 (0.00)   0.40 (0.09)       0.00 (0.00)     20.05 (4.64)         10
cover_gnn_exclude_50        50.00 (0.00)   47.90 (2.30)   0.00 (0.00)   0.74 (0.22)       0.00 (0.00)    56.61 (26.53)         10
cover_gnn_exclude_500      500.00 (0.00)   50.00 (0.00)   0.00 (0.00)   0.38 (0.06)       0.00 (0.00)     18.98 (2.77)         10
painting_gnn_exclude_100   100.00 (0.00)   30.50 (8.85)   0.00 (0.00)   2.04 (1.15)       0.00 (0.00)   254.55 (98.63)         10
painting_gnn_exclude_150   150.00 (0.00)   32.60 (5.75)   0.00 (0.00)   1.49 (0.81)       0.00 (0.00)   225.62 (60.22)         10
painting_gnn_exclude_200   200.00 (0.00)   32.70 (6.62)   0.00 (0.00)   1.62 (0.66)       0.00 (0.00)   228.62 (63.47)         10
painting_gnn_exclude_50     50.00 (0.00)  23.10 (12.95)   0.00 (0.00)   2.48 (1.44)       0.00 (0.00)  319.58 (125.70)         10
tools_gnn_exclude_100      100.00 (0.00)    4.70 (1.79)   0.00 (0.00)   0.67 (0.64)       0.00 (0.00)   410.10 (13.01)         10
tools_gnn_exclude_150      150.00 (0.00)    8.20 (3.74)   0.00 (0.00)   0.34 (0.36)       0.00 (0.00)   387.76 (50.87)         10
tools_gnn_exclude_200      200.00 (0.00)   13.10 (4.09)   0.00 (0.00)   0.39 (0.37)       0.00 (0.00)   339.54 (39.33)         10
tools_gnn_exclude_50        50.00 (0.00)    1.10 (1.87)   0.00 (0.00)   0.37 (0.17)       0.00 (0.00)   431.96 (23.78)         10
```

**Pull request merged**